runtime/compiler/env/VMJ9.cpp

```diff
@@ -6503,6 +6503,15 @@ TR_J9VMBase::isAbstractClass(TR_OpaqueClassBlock * clazzPointer)
    return (TR::Compiler->cls.romClassOf(clazzPointer)->modifiers & J9AccAbstract) ? true : false;
    }
 
+bool
+TR_J9VMBase::isConcreteClass(TR_OpaqueClassBlock * clazzPointer)
+   {
+   if ((TR::Compiler->cls.romClassOf(clazzPointer)->modifiers & J9AccInterface) || 
```

Comment:
This could be combined into one operation:
 `return (TR::Compiler->cls.romClassOf(clazzPointer)->modifiers & (J9AccAbstract | J9AccInterface)) ? false : true;`